libraries/opensk/src/ctap/storage.rs

```diff
@@ -39,6 +41,8 @@ struct PinProperties {
     hash: [u8; PIN_AUTH_LENGTH],
 
     /// Length of the current PIN in code points.
+    // This field is unused when compiled without config_command.
+    #[allow(dead_code)]
```

Comment:
This could become `[cfg_attr(not(feature = "config_command"), allow(dead_code))]` when `std` is moved out of the equation.

Reply:
Done